packages/hardhat-core/src/internal/hardhat-network/provider/modules/logger.ts

```diff
@@ -17,10 +23,18 @@ export class ModulesLogger {
     this._logs.push(message);
   }
 
-  public logWithTitle(title: string, message: string, indent = false) {
+  public setIndent(flag: boolean) {
+    this._indentEnabled = flag;
+  }
```

Comment:
The name of this method is misleading for 2 reasons:
1. There's an `_indent` field but it's not modified by this method.
2. I'd expect a `setIndent` method to take a `number` not a `boolean`.
```suggestion
  public setIndentEnabled(flag: boolean) {
    this._indentEnabled = flag;
  }
```